src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -65,34 +65,34 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
             {
                 this.meter.CreateObservableCounter($"{metricPrefix}il.bytes.jitted", () => System.Runtime.JitInfo.GetCompiledILBytes(), "By", description: "IL Bytes Jitted");
                 this.meter.CreateObservableCounter($"{metricPrefix}methods.jitted.count", () => System.Runtime.JitInfo.GetCompiledMethodCount(), description: "Number of Methods Jitted");
-                this.meter.CreateObservableGauge($"{metricPrefix}time.in.jit", () => System.Runtime.JitInfo.GetCompilationTime().TotalMilliseconds, "ms", description: "Time spent in JIT");
+                this.meter.CreateObservableGauge($"{metricPrefix}time.in.jit", () => System.Runtime.JitInfo.GetCompilationTime().Ticks * 100, "ns", description: "Time spent in JIT");
```

Comment:
Instead of using a hard-coded `100`, consider leveraging something like https://docs.microsoft.com/dotnet/api/system.timespan.tickspermillisecond.

Reply:
I create a `const` as it's the definition of ticks at https://docs.microsoft.com/dotnet/api/system.timespan.ticks#remarks and I didn't find no such field in the library.

Reply:
Sounds good, thanks!